website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,126 @@
+
+{{% alert color="info"  title="Note" %}}
+Changing the account after the creation of a cluster is not possible. The credentials can be updated with a new key or even user but have to stay within the same account. 
+
+Currently, there is no way to move a single cluster to a different account. A user would rather have to re-create a cluster and migrate workloads by different means.
+{{% /alert %}}
+
+As part of the infrastructure choice, the region for data plane has to be chosen as well. The Gardener scheduler will try to place the control plane on a seed cluster based on a minimal distance strategy. See [Gardener Scheduler](https://github.com/gardener/gardener/blob/master/docs/concepts/scheduler.md) for more details.
+
+![](./images/basic_configurations_3.png)
+
+Up next, the networking provider for the cluster has to be selected (CNI). At the point of writing it is possible to choose between Calico and Cilium. If not specified in the shoot's manifest, default CIDR ranges for nodes, services, and pods will be used.
+
+In order to run any workloads in your cluster, you need nodes. The worker section lets you specify the most important configuration options. For beginners, the machine type is probably the most relevant field, together with the machine image (operating system). 
+
+The machine type is provider-specific and configured in the cloud profile. Check your respective cloud profile if you're missing a machine type. Maybe it is available but not in this region.
+
+The operating system your machines will run is the next thing to choose. Debian-based [GardenLinux](https://github.com/gardenlinux/gardenlinux) is the best choice for most use cases.
+
+Other specifications for the workers include the volume type and size. These settings affect the root disk of each node. Therefore we would always recommend to use an SSD-based type to avoid i/o issues.
+
+The autoscaler parameter defines the initial elasticity / scalability of your cluster. The cluster-autoscaler will add more nodes up to the maximum defined here when your workload grows and remove nodes in case your workload shrinks. The minimum number of nodes should be equal to or higher than the number of zones. You can distribute the nodes of a worker pool among all zones available to your cluster. It is the first step in running HA workloads.
+
+![](./images/basic_configurations_4.png)
+
+Once per day, all clusters reconcile. This means all controllers will check if there are any updates they have to apply (i.e. new image version for ETCD). The maintenance window defines when this daily operation will be triggered. It is important to understand that there is no opt-out for reconciliation.
+
+It is also possible to confine updates to the shoot spec to be applied only during this time. This can come in handy when you want to bundle changes or prevent changes to be applied outside a well-known time window.
```

Comment:
```suggestion
It is also possible to confine updates to the Shoot spec to be applied only during this time. This can come in handy when you want to bundle changes or prevent changes to be applied outside a well-known time window.
```